pyaedt/edb.py

```diff
@@ -177,31 +181,23 @@ def open_edb(self, init_dlls=False):
         if init_dlls:
             self._init_dlls()
 
-        if not self.isreadonly:
-            print(self.edbpath)
-            print(self.edbversion)
-            print(_ironpython)
-            print(dir())
-
-            if _ironpython and  inside_desktop:
-                print("opening from DB")
-                self._db = self.edb.Database.Open(self.edbpath, self.isreadonly)
-                self._active_cell =  list(self._db.TopCircuitCells)[0]
-                self.builder = self.layout_methods.GetBuilder(self._db, self._active_cell)
-            else:
-                print("opening from EDBLib")
-                self.builder = self.layout_methods.OpenEdbStandAlone(self.edbpath, self.edbversion)
-                self._db = self.builder.EdbHandler.dB
-                self._active_cell = self.builder.EdbHandler.cell
+        print(self.edbpath)
+        print(self.edbversion)
+        print(_ironpython)
+        print(dir())
```

Comment:
We should avoid using print statements and instead use the `logging` module.  Many users will want to suppress output to stdout or will want to have a different level of logging depending on their workflow.  Using the `logging` module lets them do that.

Reply:
Done. I've replaced with messenger calls